shops/src/main/resources/shops-migrations/V16__NEW_Shops.sql

```diff
@@ -0,0 +1,156 @@
+VALUES ('Building', 'PRISMARINE', 'Prismarine', 0, 43, 1, 1, 20, 5);
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) 
+VALUES ('Building', 'SEA_LANTERN', 'Sea Lantern', 0, 44, 1, 1, 700, 350);
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) 
+VALUES ('Building', 'CLAY', 'Clay', 0, 47, 1, 1, 50, 10);
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) 
+VALUES ('Building', 'SMOOTH_SANDSTONE', 'Smooth Sandstone', 0, 51, 1, 1, 40, 10);
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) 
+VALUES ('Building', 'SAND', 'Sand', 0, 52, 1, 1, 10, 2);
+
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) 
+VALUES ('Building', 'GLASS', 'Glass', 0, 53, 1, 1, 100, 2);
+
+-- Page 4 of Building
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'GLOW_INK_SAC', 'Glow Ink', 0, 24, 4, 1, 1000, 100);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'END_ROD', 'End Rod', 0, 23, 4, 1, 1000, 100);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'GLOW_LICHEN', 'Glow Lichen', 0, 21, 4, 1, 100, 15);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'SHROOMLIGHT', 'Shroomlight', 0, 22, 4, 1, 700, 150);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'OCHRE_FROGLIGHT', 'Ochre Froglight', 0, 31, 4, 1, 700, 150);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'VERDANT_FROGLIGHT', 'Verdant Froglight', 0, 30, 4, 1, 700, 150);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'PEARLESCENT_FROGLIGHT', 'Pearlescent Froglight', 0, 32, 4, 1, 700, 150);
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Building', 'SCULK_VEIN', 'Sculk Vein', 0, 20, 4, 1, 100, 15);
+
+-- Add Copper Ores
+INSERT IGNORE INTO shopitems (Shopkeeper, Material, ItemName, ModelData, MenuSlot, MenuPage, Amount, BuyPrice, SellPrice) VALUES ('Resources', 'COPPER_INGOT', 'Copper', 0, 7, 1, 1, 100, 25);
+
+-- Fix Dye Shop
+UPDATE shopitems SET SellPrice = 0 WHERE Material LIKE '%_DYE%';
```

Comment:
Sonar Cloud does not like having a wildcard in front of _dye
Changing it to be explicit should fix this
```suggestion
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'RED_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'BROWN_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'YELLOW_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'GREEN_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'BLUE_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'PURPLE_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'BLACK_DYE';
UPDATE shopitems
    SET SellPrice = 0
    WHERE Shopkeeper = 'Building'
    AND Material = 'WHITE_DYE';
```